Jenkinsfile_CNP

```diff
@@ -52,6 +52,14 @@ def uploadCoreCaseDataDefinitions(env) {
   }
 }
 
+def uploadBpmnDiagrams(String env) {
+  dir("${WORKSPACE}/bin") {
```

Comment:
can we read a directory and upload all BPMN file present there.

Reply:
done